jcapi-tags.go

```diff
@@ -127,15 +111,16 @@ func (jc JCAPI) GetAllTags() (tagList []JCTag, err JCError) {
 }
 
 func (jc JCAPI) GetTagByName(tagName string) (tag JCTag, err JCError) {
-	url := fmt.Sprintf("/tags/%s", tagName)
+	url := fmt.Sprintf("%s/%s", TAGS_PATH, tagName)
 
-	retVal, err := jc.Get(url)
+	tags, err := jc.GetTagsByUrl(url)
 	if err != nil {
-		err = fmt.Errorf("ERROR: Could not get tag by name/id '%s', err='%s'", tagName, err)
+		err = fmt.Errorf("ERROR: Could not get tags by name for '%s', url='%s', err='%s'", tagName, url, err.Error())
+		return
 	}
 
-	if retVal != nil {
-		jc.getTagFieldsFromInterface(retVal.(map[string]interface{}), &tag)
+	if len(tags) > 0 {
+		tag = tags[0]
 	}
```

Comment:
Might want to at least log a warning if `len(tags) > 1` to help people see if/when that issue occurs.